manager-bundle/bin/contao-console

```diff
@@ -19,17 +19,15 @@ set_time_limit(0);
 @ini_set('zlib.output_compression', '0');
 @ini_set('display_errors', '0');
 
-if (file_exists(__DIR__.'/../autoload.php')) {
-    $projectDir = \dirname(__DIR__, 2);
-} elseif (file_exists(__DIR__.'/../../../autoload.php')) {
-    $projectDir = \dirname(__DIR__, 4);
-} elseif (file_exists(__DIR__.'/../../../../autoload.php')) {
-    $projectDir = \dirname(__DIR__, 5);
-} elseif (false !== ($cwd = getcwd()) && file_exists($cwd.'/vendor/autoload.php')) {
-    $projectDir = $cwd;
-} else {
-    $projectDir = \dirname(__DIR__, 4);
-}
+$cwd = getcwd();
+
+$projectDir = match (true) {
+    false !== $cwd && str_ends_with($cwd, '/tools/cypress/webspace') => $cwd,
```

Comment:
```suggestion
$projectDir = isset($_composer_autoload_path) ? dirname($_composer_autoload_path, 2) : match (true) {
```

See https://github.com/composer/composer/blob/main/doc/articles/vendor-binaries.md#finding-the-composer-autoloader-from-a-binary

We should probably do this in all binaries I think :)